python/cog/director/director.py

```diff
@@ -258,8 +265,13 @@ def _handle_message(self, message_id, message_json) -> None:
         # predictions, but isn't exiting.
         if tracker.status == schema.Status.FAILED:
             self._record_failure()
+            span.set_attribute("prediction.status", 'success')
         else:
             self._record_success()
+            span.set_attribute("prediction.status", 'failure')
+
+        time_rfc3339 = datetime.datetime.now(datetime.timezone.utc).isoformat()
+        span.set_attribute("prediction.completed_at", time_rfc3339)
```

Comment:
We're pretty scattered in our approach to timing events but I think we might be settling on millisecond precision timestamps. @nickstenning might have other thoughts!